docs/changelog.rst

```diff
@@ -2,6 +2,8 @@
 Changelog
 =========
 
+* :bug:`-` Users will now be able to decode compounding transactions for Convex gauges.
```

Comment:
Isn't this a feature? Every decoder addition we make is not a bugfix. It's still fine to put it in bugifxes branch though not really needed as we should keep it for bugs.

```suggestion
* :feature:`-` Users will now be able to decode compounding transactions for Convex gauges.
```